src/olympia/blocklist/tests/test_commands.py

```diff
@@ -37,15 +37,22 @@ def test_empty(self):
         """
         addon_factory()
         assert Block.objects.count() == 0
+        assert KintoImport.objects.count() == 0
         call_command('import_blocklist')
         assert Block.objects.count() == 0
+        assert KintoImport.objects.count() == 6
+        assert KintoImport.objects.filter(
+            outcome=KintoImport.OUTCOME_NOTFIREFOX).count() == 1
```

Comment:
How do you know to expect this? Is it just assumed that the `blocklist.json` file is set up so that one record will result in `OUTCOME_NOTFIREFOX`? Would it be possible to set this up in the test, so one can see just from looking at the test file what the valid assertions would be? Perhaps that's overly complicated and not the way these tests are generally written, but in the frontend and code-manager code bases we try as much as possible to have all of the test setup in the tests themselves so that one can see, just from the test, what the logical expectations are.

I wonder if it could be as simple as modifying the `json` after it is read from `blocklist.json`?

Reply:
The blocklist.json file is a cut down (with guids mangled for sanitation) version of the actual blocklist so I was trying to keep it as authentic as possible.  I removed a few of the unnecessary assertions though - it was redundant to check in _every_ test that there was only one `OUTCOME_NOTFIREFOX`